concepts/list-methods/about.md

```diff
@@ -132,7 +132,7 @@ If you want the sort to be in descending order, pass the `reverse=True` argument
 ["Tony", "Thor", "Natasha", "Bruce"]
 ```
 
-For cases where sorting the original list is undesirable, the built-in [`sorted()`][sorted] can be used to return a new, sorted copy of your original list.
+For cases where mutating the original list is undesirable, the built-ins [`sorted()`][sorted] and [`reversed()`][reversed] can be used.  `sorted()` will return a sorted copy, and takes the same parameters as `<list>.sort()`. `reversed()` will return a reverse _iterator_ that can be used to build a new list copy.
```

Comment:
```suggestion
For cases where mutating the original list is undesirable, the built-in functions [`sorted()`][sorted] and [`reversed()`][reversed] can be used.  `sorted()` will return a sorted copy, and takes the same parameters as `<list>.sort()`. `reversed()` will return a reverse _iterator_ that can be used to build a new list copy.
```

Reply:
Is `iterator` going to confuse students if we're assuming they don't yet know what that is?

Reply:
It might.  But I don't know how else we'd refer to it.  We can remove both that and the ref to `reversed()` -- we could always introduce it later.  You recommendation?  Would someone coming to Python from another programming language be confused by `iterator`??

Reply:
Maybe,

`reversed()` returns an _iterator_ that yields the list's items in reverse order. We'll talk about _iterators_ more in later exercises.

Reply:
Nice!  I like that.  😄